html/host.html

```diff
@@ -3,66 +3,100 @@
-          <option>[3] Bug Hunters Camouflage</option>
-          <option>[3] Disease With Android Avoidance</option>
-          <option>[3] Minority Game</option>
-          <option>[3] Tragedy of the Commons</option>
-          <option>[3] Walking</option>
-          <option>[2] Disease Doctors</option>
-          <option>[2] Memory</option>
-          <option>[2] Oil Cartel Alternate</option>
-          <option>[2] Public Good</option>
-          <option>[1] BeeSmart</option>
-          <option>[1] Bug Hunters Adaptations</option>
-          <option>[1] Bug Hunters Competition</option>
-          <option>[1] Dice Stalagmite</option>
-          <option>[1] Fish Spotters</option>
-          <option>[1] Gridlock</option>
-          <option>[1] Guppy Spots</option>
-          <option>[1] PANDA BEAR</option>
-          <option>[1] Polling</option>
-          <option>[1] Polling Advanced</option>
-          <option>[1] Prisoners Dilemma</option>
-
-          <option>[a] Oil Cartel</option> <!-- requires narrowcast plots -->
-
-          <option>[0] Restaurants</option> <!-- makes no distinction between AI restaurants and player-owned ones -->
-
-          <option>[0] Bird Breeders</option> <!-- requires spectator clients -->
-          <option>[0] Gridlock Alternate</option> <!-- requires spectator clients -->
-
-          <option>[0] Client Overrides Example</option> <!-- requires view overrides -->
-          <option>[0] Critter Designers</option> <!-- requires view overrides -->
-          <option>[0] Current in a Wire</option> <!-- requires view overrides -->
-          <option>[0] MTG 1 Equal Opportunities</option> <!-- requires view overrides -->
-          <option>[0] MTG 2 Random Assignment</option> <!-- requires view overrides -->
-          <option>[0] MTG 3 Feedback Loop</option> <!-- requires view overrides -->
-          <option>[0] Sampler</option> <!-- requires view overrides -->
-        </select>
-        <label>
-          Session Name:
-          <input name="sessionName" type="text" pattern="\w.*" required>
-        </label>
-        <label>
-          Password (optional):
-          <input name="password" type="password">
-        </label>
-        <input type="submit" value="Launch">
+
+        <!-- Select activity div -->
+        <div class="activity-selection">
+
+          <!-- Activity selection + description  -->
+          <div class="activity-description">
+            <div class="library-model-container" class="library-model-container">
+              <select class="library-model" id="library-model" name="libraryModel" type="chooser"></select>
+            </div>
+            <p id="activity-text-short" class="activity-text-short"></p>
+            <button id="read-more-btn" class="read-more-btn" type="button">Read More</button>
+          </div>
+
+          <!-- Activity image -->
+          <div class="img-wrapper">
+            <img id="activity-img" class="img" alt="Activity Image">
```

Comment:
`id="preview-image" class="preview-image"`

Reply:
Yep, done & I also changed everywhere I originally named something `img` to `image` to be more explicit and clear.